.kitchen.yml

```diff
@@ -6,6 +6,10 @@ suites:
   - name: default
     run_list:
       - recipe[osl-git::default]
+  - name: osl-gitlfs
+    run_list:
+      - recipe[osl-git::default]
```

Comment:
If you make the change I mentioned in the resource, this recipe shouldn't be included here in the suite anymore.